doc/man7/openssl-qlog.pod

```diff
@@ -0,0 +1,248 @@
+=pod
+
+=head1 NAME
+
+openssl-qlog - OpenSSL qlog tracing functionality
+
+=head1 DESCRIPTION
+
+OpenSSL 3.3 and later feature unstable support for generating logs in the
```

Comment:
This should not be noted this way - we should have the usual History section - but nothing about Version X and later - that doesn't fit here.

e.g.

"OpenSSL includes support for generating logs in the qlog format, which ... "

And then 

History:

These functions were added in OpenSSL 3.3.

(or should it say feature/capability/option rather than "functions"

Reply:
Good catch.